turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksSession.java

```diff
@@ -8,8 +8,6 @@
 import android.graphics.drawable.ColorDrawable;
 import android.graphics.drawable.Drawable;
 import android.os.Build;
-import androidx.annotation.RequiresApi;
-import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
```

Comment:
Why was line 12 removed? Unused?

Reply:
Android Studio will "optimize imports", so `RequiresApi` got moved down and yea `SwipeRefreshLayout` isn't used in this class.